sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -420,13 +420,15 @@ case class DeleteAction(condition: Option[Expression]) extends MergeAction
 
 case class UpdateAction(
     condition: Option[Expression],
-    assignments: Seq[Assignment]) extends MergeAction {
+    assignments: Seq[Assignment],
+    assignStar: Boolean) extends MergeAction {
   override def children: Seq[Expression] = condition.toSeq ++ assignments
 }
 
 case class InsertAction(
     condition: Option[Expression],
-    assignments: Seq[Assignment]) extends MergeAction {
+    assignments: Seq[Assignment],
+    assignStar: Boolean) extends MergeAction {
```

Comment:
shall we simply add a new implementation of `MergeAction`: `AssignStarAction`?

Reply:
in that case shouldnt it be `InsertStarAction(condition)` and `UpdateStarAction(condition)`?